.github/workflows/cd.yml

```diff
@@ -148,28 +148,6 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.CD_PAT }}
 
-      - name: release templates' alpha version to github
-        if: ${{ contains(steps.version-change.outputs.CHANGED, 'templates@') && github.ref == 'refs/heads/dev' }}
-        uses: ncipollo/release-action@v1.10.0
-        with:
-          token: ${{ secrets.CD_PAT }}
-          prerelease: true
-          tag: "templates-0.0.0-alpha"
-          artifacts: ${{ runner.temp }}/teamsfx_templates/*.zip, ${{ runner.temp }}/teamsfx_templates_v3/*.zip
-          allowUpdates: true
-          removeArtifacts: true
-
-      - name: release templates' prerelease version to github
```

Comment:
it's better to keep this step.
we will keep the release branch afterwards. The stable branch launch is more stable and the cycle may be longer. 
Instead of the prerelease branch, there will be a 2-weekly.

Reply:
Updated.